extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts

```diff
@@ -75,15 +74,16 @@ export class InstanceSummary {
 		return instanceSummaryContainer;
 	}
 
-	public populateInstanceSummaryContainer(model: MigrationStateModel, targetType: MigrationTargetType): void {
+	// function to populate instance summary container with latest values.
+	public populateInstanceSummaryContainer(model: MigrationStateModel): void {
 
 		this._assessedDatabases.value = constants.ASSESSED_DBS_LABEL + ": " + model._databasesForAssessment?.length;
-		this._totalFindingLabels.value = constants.TOTAL_FINDINGS_LABEL + ": " + model._assessmentResults?.issues.filter(issue => issue.appliesToMigrationTargetPlatform === targetType).length;
-		const readyDbsCount = model._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === targetType).length === 0).length;
+		this._totalFindingLabels.value = constants.TOTAL_FINDINGS_LABEL + ": " + model._assessmentResults?.issues.filter(issue => issue.appliesToMigrationTargetPlatform === model._targetType).length;
+		const readyDbsCount = model._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === model._targetType).length === 0).length;
 		const notReadyDbsCount = model._databasesForAssessment?.length - readyDbsCount;
 		const readywithWarnDbsCount = 0;
```

Comment:
For ReadyWithWarnings, we can use IssueCategory. So sql nuget sends two type of issues - issue and warning. 
if issue with issueCategory = "Issue" exists in database, it is notReady.
if issues with only issueCategory="Warning" exist, it is ready with warnings.
Else it is ready.

Reply:
yes this is a good catch, let me correct this one.